internal/utils/gcs/gcs_utils.go

```diff
@@ -37,30 +38,30 @@ func uploadToGCS(ctx context.Context, client *storage.Client, bucket, filename s
 	return fmt.Sprintf("gs://%s/%s", bucket, objectName), nil
 }
 
-func deleteFromGCS(ctx context.Context, client *storage.Client, bucket, objectName string) error {
+func DeleteFromGCS(ctx context.Context, client *storage.Client, bucket, objectName string) error {
 	if err := client.Bucket(bucket).Object(objectName).Delete(ctx); err != nil {
 		return fmt.Errorf("failed to delete object from GCS: %w", err)
 	}
 	return nil
 }
 
-func createStorageClient(ctx context.Context, opts TranscribeOptions) (*storage.Client, error) {
+func CreateStorageClient(ctx context.Context, opts common.TranscribeOptions) (*storage.Client, error) {
```

Comment:
I would have used


```suggestion
func NewStorageClient(ctx context.Context, opts common.TranscribeOptions) (*storage.Client, error) {
```

It sounds more usual for a constructor